### ISSUE TITLE
make sure the transformation process of pre-processed data is saved to be used on test set and future predictions

### ISSUE THREAD

**Issue by shane-kercheval** [shane-kercheval/python-predictive-analysis-template#1]
https://stackoverflow.com/questions/38780302/predicting-new-data-using-sklearn-after-standardizing-the-training-data
